sysconfig/sysconfig.go

```diff
@@ -41,3 +43,9 @@ func ConfigureRunSystem(opts *Options) error {
 
 	return nil
 }
+
+// WritableDefaultsDir returns full path of subdir under
+// rootdir/_writable_defaults.
+func WritableDefaultsDir(rootdir, subdir string) string {
```

Comment:
my idea was to make this take `subdir ...string`

Reply:
Ah, ok, thanks